lib/usb_hid_host/usb_hid_host.h

```diff
@@ -21,10 +19,16 @@ typedef struct {
     int8_t scroll_wheel;
 } __attribute__((packed)) unified_mouseReport_t;
 
+// Callback function pointer for applications to receive mouse reports
 typedef void (*mouse_report_callback_t)(unified_mouseReport_t* mouse_report);
 
 void register_mouse_report_callback(mouse_report_callback_t callback);
-void unbond_all_devices();
+mouse_report_callback_t * get_registered_mouse_callback();
```

Comment:
Better to have all mouse functions &  callbacks in usb_hid_mouse.h?

Reply:
the thing is that this is actually a "unified mouse data" callback, which is also used e.g. for joystick reports (there, the first stick is mapped to x/y mouse). we could use different callbacks for the different hid devices and treat their data differntly in the main application code... however the questions remains what data to use e.g. from a joystick with multiple axis and many buttons... For now, i'd keep it simple like it is - but we could change the name of the callback (e.g. to "unified_hidData_callback?) to avoid confusion.